### PR TITLE
[NLU-3222] V3 Entity Improvement: EN-US time: 10 past 3

### PR DESCRIPTION
**Pull request by andrew-gradinari**
This is a fix for the "[minutes] past [hour]" time format such as "10 past 3" and "ten past three".

### CODE REVIEW

**Comment by andrew-gradinari** [comment]
Sample pipeline run: https://jenkins.ininica.com/job/NLU/job/Recognizers-Text/181/

File: Patterns/English/English-DateTime.yaml [hunk 1]
```diff
@@ -362,7 +362,7 @@ NightRegex: !simpleRegex
 CommonDatePrefixRegex: !simpleRegex
   def: ^[\.]
 LessThanOneHour: !nestedRegex
-  def: (?<lth>(a\s+)?quarter|three quarter(s)?|half( an hour)?|{BaseDateTime.DeltaMinuteRegex}(\s+(minutes?|mins?))|{DeltaMinuteNumRegex}(\s+(minutes?|mins?)))
+  def: (?<lth>(a\s+)?quarter|three quarter(s)?|half( an hour)?|{BaseDateTime.DeltaMinuteRegex}(\s+(minutes?|mins?)|(?=\s+past))|{DeltaMinuteNumRegex}(\s+(minutes?|mins?)|(?=\s+past)))
```
Comment (srandal) [inline Patterns/English/English-DateTime.yaml:365]: I didn't know it was possible to use a look-ahead expression in alternation with matching text, but this makes sense.

**Review by srandal — APPROVED** [review]

**Comment by samhickey25** [comment]
Are there any existing tests which were skipped that can be unskipped now?

**Review by samhickey25 — APPROVED** [review]

**Review by kevinwalshgen — APPROVED** [review]

**Review by colm-dillon — APPROVED** [review]

**Review by sanhal — APPROVED** [review]

**Comment by andrew-gradinari** [comment]
> Are there any existing tests which were skipped that can be unskipped now?

I have not found any existing tests for this case.

**Pull request merged**